pkg/config/config.go

```diff
@@ -216,12 +217,12 @@ func askForLoadBalancer() (string, error) {
 
 // renderMultiNodeConfig renders a configuration to allow k0sctl to install in a multi-node
 // configuration.
-func renderMultiNodeConfig(ctx context.Context, nodes []infra.Node) (*v1beta1.Cluster, error) {
+func renderMultiNodeConfig(ctx *cli.Context, nodes []infra.Node, helmConfigs dig.Mapping) (*v1beta1.Cluster, error) {
```

Comment:
can we use another variable name for the cli.Context ? In some other places here it is called only `c`.  BTW, why do we need to pass in the urfave type this deep into the code path ? Consider we might want to migrate to cobra later on.